### PR TITLE
create readme

### PR DESCRIPTION
**Pull request by Panchenko-Volodymyr**

### CODE REVIEW

**Review by zachar-chorny — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,52 @@
-# TAXI
\ No newline at end of file
+# Taxi Service
+
+Do you have taxi business or want some?
+So you need this app!
+
+Taxi Servise help you with organization of your workers and cars.
+Application is easy to use and very understandable.
+![taxi](https://image.freepik.com/free-vector/taxi-on-the-city_1270-526.jpg)
```
Comment (Nick97-git) [inline README.md:9]: move image to the next line

File: README.md [hunk 2]
```diff
@@ -1 +1,52 @@
...
+- Service can add drivers who drive one car, as well as cars that are driven by one driver
+- See all created cars, manufacturers or drivers.
+- Work with features this app access can only for registered drivers
+- In the development we used 3-layered architecture, that means if we need to expand our features It will be easy to implement
+***
+### Technologies which was used in app :
+* Apache Tomcat (v9.0.56)
+* Servlet
+* JDBC
+* MySQL, MySQL Workbench 8.0
```
Comment (Nick97-git) [inline README.md]:
```suggestion
* MySQL
```

File: README.md [hunk 3]
```diff
@@ -1 +1,52 @@
...
+    * Paste it in Workbench
+    * Refresh All
+4. In `src/main/java/mate/util/ConnectionUtil.class` we need to do some changes, and replace with yours :
+    * USERNAME and PASSWORD
+    * URL and JDBC_DRIVER
+   ```
+     //example
+     private static final String URL = "jdbc:mysql://localhost:3306/taxi";
+     private static final String JDBC_DRIVER = "com.mysql.cj.jdbc.Driver";
+
```
Comment (Nick97-git) [inline README.md]: what about USERNAME and PASSWORD examples?

File: README.md [hunk 4]
```diff
@@ -1 +1,52 @@
...
+***
+### Things that we need to do for correct working application:
+1. Firstly we need to install our additional applications. Such as :
+    * Apache Tomcat (v9.0.56)
+    * MySQL, MySQL Workbench 8.0
+2. Configure our Tomcat in IDE
+    * Click to Edit/Add configure
+    * Find Tomcat
+    * Click to red bulb
+    * Chose war exploded
```
Comment (Nick97-git) [inline README.md]:
```suggestion
2. Configure Tomcat in IDE
```

File: README.md [hunk 5]
```diff
@@ -1 +1,52 @@
...
+3. Create a schema in MySQL Workbench using file `src/main/resources/init_db.sql`:
+    * Copy all from file
+    * ALERT If you already have database, which named "taxi", this script can drop it.
+    * Paste it in Workbench
+    * Refresh All
```
Comment (Nick97-git) [inline README.md]:
```suggestion
3. Create a schema in MySQL Workbench using file `src/main/resources/init_db.sql`:
```

**Review by Nick97-git — CHANGES_REQUESTED** [review]
- Remove redundant / from endpoints like /cars/, /drivers/ and /manufacturers/ from all places where you use them
- DeleteCarController, DeleteDriverController, DeleteManufacturerController -> remove /all from /cars/all, /drivers/all, /manufacturers/all url patterns

File: README.md [hunk 6]
```diff
@@ -1 +1,56 @@
...
+   ```
+   //example
+   private static final String USERNAME = "root";
+   private static final String Password = "qwerty";
+   ```
+    * URL and JDBC_DRIVER
+   ```
+     //example
+     private static final String URL = "jdbc:mysql://localhost:3306/taxi";
+     private static final String JDBC_DRIVER = "com.mysql.cj.jdbc.Driver";
```
Comment (Nick97-git) [inline README.md]: unite them in one

**Comment by Nick97-git** [comment]
also, you didn't make changes according to the comment about redundant `/`

**Review by Nick97-git — CHANGES_REQUESTED** [review]

**Review by Nick97-git — APPROVED** [review]

**Pull request closed without merging**